target/linux/realtek/image/Makefile

```diff
@@ -85,6 +103,7 @@ define Build/linksys-image
 endef
 
 define Device/Default
+  LZMA_TEXT_START := 0x82000000
```

Comment:
What are the typical sizes of the compressed (loader + payload) and decompressed (payload) blobs? If an uncompressed payload is already close to 32MB, might it make sense to move this to e.g. 48MB?

Reply:
Simply going with the "standard".

```
[markus@openwrt linux]$ grep KERNEL_LOADADDR */image/Makefile | grep " := " | wc -l
18
[markus@openwrt linux]$ grep KERNEL_LOADADDR */image/Makefile | grep " = " | wc -l
4
```

Reply:
No real idea. just following the largest value of other targets here:

```
[openwrt linux]$ grep LZMA_TEXT_START  */image/Makefile | grep 0x
ath79/image/Makefile:  LZMA_TEXT_START := 0x81800000
bmips/image/Makefile:LZMA_TEXT_START := 0x82000000              # RAM start + 32M
lantiq/image/Makefile:          LZMA_TEXT_START=0x82000000 \
ramips/image/Makefile:  LZMA_TEXT_START := 0x81800000
realtek/image/Makefile:  LZMA_TEXT_START := 0x82000000

```

Reply:
Fine with that?